### DISCUSSION TITLE
Cannot find module '@capacitor/camera' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by gcarrillo-wx** [quasarframework/quasar#10959] [CLI - Capacitor / Cordova modes] [answered]
Hi! i have this problem when i ran quasar -m capacitor -T android

src-capacitor > package.json > dependencies

`  "dependencies": {
    "@capacitor/android": "2.5.0",
    "@capacitor/camera": "^1.1.0",
    "@capacitor/cli": "^2.0.0",
    "@capacitor/core": "^2.0.0"
  }`

Console: 

`export 'CapacitorException' (imported as 'CapacitorException') was not found in '@capacitor/core'....`

Quasar Info:

`Operating System - Linux(5.4.0-88-generic) - linux/x64
NodeJs - 14.17.6

Global packages
  NPM - 6.14.15
  yarn - 1.22.11
  @quasar/cli - 1.2.1
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.1.0 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.1.0 -- Quasar Framework local CLI
  @quasar/extras - 1.11.0 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.4 -- The progressive JavaScript framework for buiding modern web UI.
  vue-router - 4.0.11
  vuex - 4.0.2 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.15.5 -- Babel compiler core.
  webpack - 5.53.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.0.0 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.2.2 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - 2.5.0 -- Capacitor: cross-platform mobile apps with the web
  @capacitor/cli - 2.5.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/android - 2.5.0 -- Capacitor: cross-platform mobile apps with the web
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*`

**Comment by hawkeye64** [comment]
I'm not sure, but I think you are looking for this: https://www.npmjs.com/package/@types/fs-capacitor

**Comment by rstoenescu** [comment, marked as answer]
You're using Capacitor v2 and reading from Capacitor v3 docs.
This is the link for Capacitor v2: https://capacitorjs.com/docs/v2/apis/camera#camera

```js
import { Plugins, CameraResultType } from '@capacitor/core'
const { Camera } = Plugins
```
